frontend/src/components/ui/blockEditor/ComputationsFileEditor.jsx

```diff
@@ -138,16 +128,22 @@ export default function ComputationsFileEditor({ fetchFileSystem }) {
     setIsLoading(false);
   };
 
-  const handleSave = (e) => {
+  useEffect(() => {
+    if (editorRef.current) {
+      setTimeout(() => {
+        editorRef.current.scrollIntoView({
+          behavior: "smooth",
+          block: "start",
+        });
+      }, 200);
```

Comment:
Using a timeout is a "hack" since it might fail if the file is very long and code mirror takes more than 200 milliseconds to process the content. Code mirror has an `onUpdate` function, and I think that's where the code should be to make it more robust. `TestLogs.jsx` Does something like that

Reply:
Isn't `onUpdate` called when the code inside of it is edited?

Reply:
Oh yeah, that might be the case. We'd have to find a way to make the trigger scroll on the first call to `onUpdate` only then.